src/components/scoresheets/entry.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import './entry.css';
+
+//=========================================
+// props / actions
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    entry: ownProps.entry,
+    scoring: ownProps.entry.scoring,
+    country: ownProps.country,
+  };
+};
+
+const mapDispatchToProps = (dispatch) => ({
+
+});
+
+//=========================================
+// component
+
+class Entry extends React.Component {
```

Comment:
I have an entryShow component, there might be a way to re-use these somehow. Would this be better titled something more than just entry since we have displaying entry in multiple ways? Also does it make sense to keep this file in the entries components folder?